ethpm/utils/ipfs.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from typing import (
+    Any,
+    Dict,
+)
+
+from urllib import (
+    parse,
+    request,
+)
+
+from eth_utils import to_text
+
+
+INFURA_IPFS_URI_PREFIX = "https://ipfs.infura.io/ipfs/"
+
+
+def fetch_ipfs_package(hash: str) -> Dict[str, Any]:
+    package_uri = INFURA_IPFS_URI_PREFIX + hash
+    with request.urlopen(package_uri) as response:
+        package_data = response.read()
+    return json.loads(to_text(package_data))
```

Comment:
I think you can get this as `response.json()`

You also *probably* should:

1. call `response.raise_for_status()` to take care of error http responses.
2. not MVP but worth tracking as an issue and maybe commenting in the code as well, this should eventually be turned into a swappable *backend* system so that people can use something other than the infura IPFS gateway.